docs/index.rst

```diff
@@ -3,50 +3,175 @@
+
+Future releases of `pAPRika` will include support for other MD engines like ,
+`NAMD <https://www.ks.uiuc.edu/Research/namd/>`_ and `LAMMPS <https://lammps.sandia.gov/>`_. Plumed is supported by all
+of these but for NAMD only NPT simulations can be performed. Hence, a module converting `pAPRika` restraints to a
+`Colvars <https://github.com/Colvars/colvars>`_ module, which is supported in NAMD and LAMMPS, is also in the works.
+
+.. table:: MD engines that are supported in `pAPRika` (or planned) and the respective restraint modules it supports.
+   :widths: auto
+   :align: center
+   :class: clean-table property-table
+
+   +-----------------+---------------------------------------------------------------------+
+   ||                || Restraints Module                                                  |
+   || MD Engine      +----------------+----------------+-----------------+-----------------+
+   ||                || NMR           || XML           || Plumed         || Colvars\ |ast| |
+   +=================+================+================+=================+=================+
+   || AMBER          || |tick|        || |cross|       || |tick|         || |cross|        |
+   +-----------------+----------------+----------------+-----------------+-----------------+
+   || OpenMM         || |cross|       || |tick|        || |tick|\ |dast| || |cross|        |
+   +-----------------+----------------+----------------+-----------------+-----------------+
+   || GROMACS        || |cross|       || |cross|       || |tick|         || |cross|        |
+   +-----------------+----------------+----------------+-----------------+-----------------+
+   || NAMD\ |ast|    || |cross|       || |cross|       || |tick|         || |tick|         |
+   +-----------------+----------------+----------------+-----------------+-----------------+
+   || LAMMPS\ |ast|  || |cross|       || |cross|       || |tick|         || |tick|         |
+   +-----------------+----------------+----------------+-----------------+-----------------+
+
+*\* Currently not supported but will be available in future releases.* |br|
+*\*\* Supported but have not yet been tested.*
+
+===================
+
+****************
+pAPRika Workflow
+****************
+todo: workflow-diagram (explain the caveats when using different MD engines).
+
+
+===================
+
+
+**********
+References
+**********
+1. Velez-Vega, C. & Gilson, M. K. Overcoming Dissipation in the Calculation of Standard Binding Free Energies by Ligand Extraction. J. Comput. Chem. 34, 2360–2371 (2013).
+2. Fenley, A. T., Henriksen, N. M., Muddana, H. S. & Gilson, M. K. Bridging calorimetry and simulation through precise calculations of cucurbituril-guest binding enthalpies. J. Chem. Theory Comput. 10, 4069–4078 (2014).
+3. Henriksen, N. M., Fenley, A. T. & Gilson, M. K. Computational Calorimetry: High-Precision Calculation of Host−Guest Binding Thermodynamics. J. Comput. Theory Nanosci. 11, 4377–4394 (2015).
```

Comment:
Should be JCTC. Might be my typo somewhere else.

Reply:
Yes! It was recorded as `J. Comput. Theory Nanosci.` in my Mendeley library.